go/cli/mcap/cmd/doctor.go

```diff
@@ -369,6 +381,15 @@ func (doctor *mcapDoctor) Examine() {
 			doctor.error("Chunk at offset %d has uncompressed size %d, but its chunk index has uncompressed size %d", chunkOffset, chunk.UncompressedSize, chunkIndex.UncompressedSize)
 		}
 	}
+
+	if doctor.statistics != nil {
```

Comment:
is it possible we will hit this on an unchunked file, without erroring in some other way?

Reply:
cause if the file were unchunked but had statistics, I think the logic would be wrong

Reply:
what would be the right thing to do in that case? Would it work if I switch this to calculate minStartTime/maxEndTime from looking at all the message records rather than the chunk indexes?